system-tests/lib/cre/don/jobs/consensus/consensus.go

```diff
@@ -102,3 +107,33 @@ func GenerateJobSpecs(donTopology *cre.DonTopology, ds datastore.DataStore, chai
 
 	return donToJobSpecs, nil
 }
+
+var ConsensusV2JobSpecFactoryFn = func(logger zerolog.Logger, chainID uint64, config map[string]any,
+	capabilitiesAwareNodeSets []*cre.CapabilitiesAwareNodeSet,
+	infraInput infra.Input,
+	evmBinaryPath string) cre.JobSpecFactoryFn {
+	return func(input *cre.JobSpecFactoryInput) (cre.DonsToJobSpecs, error) {
+		configGen := func(nodeAddress string) (string, error) {
+			return fmt.Sprintf(
+				`'{"chainId":%d,"network":"evm","nodeAddress":"%s"}'`,
+				chainID,
+				nodeAddress,
+			), nil
+		}
+
+		return ocr.GenerateJobSpecsForStandardCapabilityWithOCR(
+			logger,
+			input.DonTopology,
+			input.CldEnvironment.DataStore,
+			chainID,
+			"evm",
```

Comment:
could it ever be anything else than `evm`?

Reply:
You're right, removing.